src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
-body {
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  width: 200px;
+  height: 408px;
```

Comment:
Avoid using fixed widths and heights for elements. This can make the design less responsive and harder to maintain. Consider using max-width or min-width for width, and padding or min-height for height, to allow for more flexibility.